pkg/iostreams/iostreams.go

```diff
@@ -466,10 +466,10 @@ func Test() (*IOStreams, *bytes.Buffer, *bytes.Buffer, *bytes.Buffer) {
 }
 
 func isTerminal(f *os.File) bool {
-	return ghTerm.IsTerminal(f) || isCygwinTerminal(f.Fd())
+	return ghTerm.IsTerminal(f) || IsCygwinTerminal(f.Fd())
 }
 
-func isCygwinTerminal(fd uintptr) bool {
+func IsCygwinTerminal(fd uintptr) bool {
```

Comment:
Since we're exporting a new function, maybe it's better to add it as a method to IOStreams? That way an IOStreams instance can just figure out the file descriptor bit without the caller having to pass it in.